instrumentation/rediscala-1.8/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/rediscala/RediscalaClientTest.java

```diff
@@ -0,0 +1,177 @@
+      system = clazz.getMethod("create").invoke(null);
+    }
+
+    try {
+      RedisClient.class.getMethod("username");
+      redisClient =
+          (RedisClient)
+              RedisClient.class.getConstructors()[0].newInstance(
+                  host,
+                  port,
+                  Option.apply(null),
+                  Option.apply(null),
+                  Option.apply(null),
+                  "RedisClient",
+                  Option.apply(null),
+                  system,
+                  new RedisDispatcher("rediscala.rediscala-client-worker-dispatcher"));
+    } catch (Exception e) {
+      redisClient =
+          (RedisClient)
+              RedisClient.class.getConstructors()[0].newInstance(
+                  host,
+                  port,
+                  Option.apply(null),
+                  Option.apply(null),
+                  "RedisClient",
+                  Option.apply(null),
+                  system,
+                  new RedisDispatcher("rediscala.rediscala-client-worker-dispatcher"));
+    }
+  }
+
+  @AfterAll
+  static void tearDown() throws Exception {
+    redisServer.stop();
+    if (system != null) {
+      system.getClass().getMethod("terminate").invoke(system);
+    }
+  }
+
+  @Test
+  void testGetCommand() {
+    Pair<Future<Object>, Future<Option<String>>> result =
+        testing.runWithSpan(
+            "parent",
+            () -> {
+              ByteStringSerializer<String> serializer = ByteStringSerializer$.MODULE$.String();
+              ByteStringDeserializer<String> deserializer =
+                  ByteStringDeserializer$.MODULE$.String();
+              Future<Object> writeFuture =
+                  redisClient.set(
+                      "bar",
+                      "baz",
+                      Option.apply(null),
+                      Option.apply(null),
+                      false,
+                      false,
+                      serializer);
+              Future<Option<String>> valueFuture = redisClient.get("bar", deserializer);
+              return Pair.of(writeFuture, valueFuture);
+            });
+
+    await().atMost(java.time.Duration.ofSeconds(3)).until(() -> result.getLeft().isCompleted());
+    await().atMost(java.time.Duration.ofSeconds(3)).until(() -> result.getRight().isCompleted());
+    assertThat(result.getLeft().value().get().get()).isEqualTo(true);
+    assertThat(result.getRight().value().get().get().get()).isEqualTo("baz");
+
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span -> span.hasName("parent").hasNoParent(),
+                span ->
+                    span.hasName("SET")
+                        .hasKind(CLIENT)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(DbIncubatingAttributes.DB_SYSTEM, "redis"),
```

Comment:
could you add a static import for `DB_SYSTEM`

Reply:
also could use `DbIncubatingAttributes .DbIncubatingAttributes.REDIS`

Reply:
done